basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/authentication/DefaultAuthenticationContextTestCase.java

```diff
@@ -0,0 +1,104 @@
+package org.jboss.hal.testsuite.test.configuration.elytron.authentication;
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.page.config.elytron.ElytronAuthenticationPage;
+import org.jboss.hal.testsuite.test.configuration.elytron.AbstractElytronTestCase;
+import org.jboss.hal.testsuite.test.configuration.elytron.ElytronOperations;
+import org.jboss.hal.testsuite.util.ConfigChecker;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.foundation.online.SnapshotBackup;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+
+import java.io.IOException;
+
+@RunWith(Arquillian.class)
+public class DefaultAuthenticationContextTestCase extends AbstractElytronTestCase {
+
+    @Page
+    private ElytronAuthenticationPage page;
+
+    private static final String
+        AUTHENTICATION_CONTEXT = "authentication-context",
+        DEFAULT_AUTHENTICATION_CONTEXT = "default-authentication-context",
+        FINAL_PROVIDERS = "final-providers",
+        INITIAL_PROVIDERS = "initial-providers";
+
+    /**
+     * @tpTestDetails Try to edit default-authentication-context attribute value in Web Console's Elytron subsystem
+     * configuration.
+     * Validate edited attribute value in the model.
+     */
+    @Test
+    public void editDefaultAuthenticationContext() throws Exception {
+        final Address authenticationContextAddress = createAuthenticationContext();
+
+        final SnapshotBackup snapshotBackup = new SnapshotBackup();
+        client.apply(snapshotBackup.backup());
```

Comment:
That's definitely better! We can call backup once in `BeforeClass` and then restore in `After`.